examples/ticks_and_spines/date_concise_formatter.py

```diff
@@ -0,0 +1,182 @@
+"""
+================================================
+Formatting date ticks using ConciseDateFormatter
+================================================
+
+Finding good tick values and formatting the ticks for an axis that
+has date data is often a challenge.  `~.dates.ConciseDateFormatter` is
+meant to improve the strings chosen for the ticklabels, and to minimize
+the strings used in those tick labels as much as possible.
+
+.. note::
+
+    This formatter is a candidate to become the default date tick formatter
+    in future versions of Matplotlib.  Please report any issues or
+    suggestions for improvement to the github repository or mailing list.
+
+First, this is the default formatter.
```

Comment:
Can this line be put after imports&data and right before the figure? It seems a bit odd here by itself.